### PR TITLE
move docs to karmada website

### LINKED ISSUE #2316: Move docs to Karmada website
**What would you like to be added**:
We will move our docs to Karmada website to better manage documents and give users a better experience to understand the Karmada project for uppon release 1.3. Now the documents are tidy ready and lately we will check the PRs about documents

**Why is this needed**:
To better manage documents and give users a better experience to understand the Karmada project.

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Remove docs to website.

**Which issue(s) this PR fixes**:
Fixes #2316 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,44 +1 @@
...
-## Reference
-
-## Troubleshooting
-Refer to [Troubleshooting](./troubleshooting.md)
-
-## Frequently Asked Questions
-
-Refer to [FAQ](./frequently-asked-questions.md).
\ No newline at end of file
+All documentations for the Karmada is hosted at [karmada-io/website](https://github.com/karmada-io/website).
```
Comment (Tingtal) [inline docs/README.md:1]:
```suggestion
Karmada documentation is all hosted on [karmada-io/website](https://github.com/karmada-io/website).
```

**Review by Tingtal — CHANGES_REQUESTED** [review]
One small touch on the language. Pls check. Thanks.

File: docs/README.md (hunk 1 above)
Comment (Poor12) [reply docs/README.md]: Done.

**Comment by Poor12** [comment]
cc @RainbowMango

**Pull request merged**